PBRVulkan/Assets/Scenes/cornell_box.scene

```diff
@@ -61,15 +56,15 @@ mesh
 mesh
 {
 	file cornell_box/cbox_smallbox.obj
-	material orange
+	material white
```

Comment:
Why do you change the scenes? As far as I understand you mention we should not do it.

Reply:
Yes, but this is the original form the file. I restored it.